.github/CODEOWNERS

```diff
@@ -103,6 +103,12 @@
 /kernel/native*   @coq/vm-native-maintainers
 /kernel/vm*       @coq/vm-native-maintainers
 /kernel/vconv.*   @coq/vm-native-maintainers
+/kernel/cbytecodes.* @coq/vm-native-maintainers
+/kernel/copcodes.* @coq/vm-native-maintainers
+/kernel/cemitcodes.* @coq/vm-native-maintainers
+/kernel/clambda.* @coq/vm-native-maintainers
+/kernel/cbytegen.* @coq/vm-native-maintainers
+/kernel/csymtable.* @coq/vm-native-maintainers
```

Comment:
Alternatively maybe we should rename those modules to Vmsomething instead of Csomething.

Reply:
I'll make a competing PR.

Reply:
See https://github.com/coq/coq/pull/12849